### PR TITLE
pylint cleanup

### PR DESCRIPTION
**Pull request by KSchopmeyer**
Removed a number of issues found by pylint including:

1. The issue with unnecessary else. Changed most of those to use the
ternary form

2. The len(condition) issue which can be generally replaced with 

    if blah:
for cases were something must exist in iterable and 

    if not blah:
if the interable should be empty or None.

3. Couple of pylint messages that pylint is now marking as obsolete.

### CODE REVIEW

File: pywbem/tupleparse.py [hunk 1]
```diff
@@ -1159,7 +1156,7 @@ def parse_property_reference(tup_tree):
 
     value = list_of_matching(tup_tree, ['VALUE.REFERENCE'])
 
-    if value is None or len(value) == 0:
+    if value is None or not value:
```
Comment (andy-maier) [inline pywbem/tupleparse.py]: Line 1159 could be simplified to:
```
if not value:
```

File: testsuite/run_cim_operations.py [hunk 2]
```diff
@@ -331,7 +320,7 @@ def assertInstancesValid(self, instances, includes_path=True,
 
             if includes_path:
                 self.assertTrue(isinstance(instance.path, CIMInstanceName))
-                self.assertTrue(len(instance.path.namespace) > 0)
+                self.assertTrue(instance.path.namespace, 'Includes namespace')
```
Comment (andy-maier) [inline testsuite/run_cim_operations.py]: The assertion message should say what the issue is when the assertion fails, not what it asserts.
I.e. change 'Includes namespace' to 'Namespace missing in instance path'.
Apart from that, is such a message really useful given that the test output shows the source line with the assertion?

File: testsuite/run_cim_operations.py [hunk 3]
```diff
@@ -1150,7 +1138,7 @@ def test_close_invalid(self):
     def test_get_onebyone(self):
         """Get instances with MaxObjectCount = 1).
         This test is subject to differences between the pull and non
-        pull sequence because it is getting live instances from the
+    gy s    pull sequence because it is getting live instances from the
```
Comment (andy-maier) [inline testsuite/run_cim_operations.py]: Remove spurious text: "gy s" ?

File: testsuite/run_cim_operations.py (hunk 2 above)
Comment (KSchopmeyer) [reply testsuite/run_cim_operations.py]: Fixed

File: testsuite/run_cim_operations.py (hunk 3 above)
Comment (KSchopmeyer) [reply testsuite/run_cim_operations.py]: Fixed

**Comment by KSchopmeyer** [comment]
Comments fixed.

**Pull request merged**